server.py

```diff
@@ -16,12 +16,16 @@ def get_next_message():
 	# assuming we are storing messages in a redis list called messages
 	# rpop removes right-end of list (least-recently added, queue)
 	new_message = r.rpop('messages')
+	resend_old_message = False
 
 	if new_message==None:
 		new_message = r.get('lastmessage')
+		resend_old_message = True
 
 	r.set('lastmessage', new_message)
-	return jsonify(json.loads(new_message.decode(encoding='UTF-8')))
+	new_message_json = json.loads(new_message.decode(encoding='UTF-8'))
+	new_message_json['resend_old_message'] = resend_old_message
+	return jsonify(new_message_json)
```

Comment:
No, if there's  no new message it pulls the last one from redis and sends it again like:
`{"message":"whatever the old message was", "size":0, "resend_old_message":true}`